Issue 695 | Improve documentation + k8s workload improvement (#765)
binbashar/le-tf-infra-aws: apps-devstg/us-east-1/k8s-eks-demoapps/README.md

    2. In the same `locals.tf` file, there is a "VPC Peerings" section.
       1. Make sure it contains the right entries to match the VPC peerings that you actually need to set up.
    3. In the `variables.tf` file you will find several variables you can use to configure multiple settings.

>        1. For instance, if you anticipate this cluster is going to be permanent, you could set the `vpc_enable_nat_gateway` flag to `true` (you can also do it updating the `terraform.tfvars` in the same folder);

Reviewer: ```suggestion
       1. For instance, if you anticipate this cluster is going to be permanent, you could set the `vpc_enable_nat_gateway` flag to `true` (you could also do that by updating the `terraform.tfvars` in the same folder);
```
